build_projects/dotnet-host-build/PublishTargets.cs

```diff
@@ -62,9 +62,13 @@ public static BuildTargetResult Publish(BuildTargetContext c)
         [BuildPlatforms(BuildPlatform.Ubuntu, "14.04")]
```

Comment:
Good call.  It looks like that works.  BuildArchitectures uses this code:

https://github.com/dotnet/core-setup/blob/1dfa45251d6706c046dff384dc81e76161f16440/build_projects/Microsoft.DotNet.Cli.Build.Framework/CurrentArchitecture.cs

Which checks the TARGETPLATFORM environment variable.

We use the same pattern on the GenerateDebs target...